lua/nvim-tree/utils.lua

```
@@ -210,6 +210,13 @@ function M.canonical_path(path)
   return path
 end
 
+-- Escapes special characters in string if windows else returns unmodified string.
+-- @param string string
+-- @return string
+function M.escape_windows_special_chars(string)
+  return M.is_windows and string:gsub("%(", "\\("):gsub("%)", "\\)") or string
```

Please guard against nil path: `string.gsub` will fail hard on nil.

---

Added nil check.